compiler/src/iree/compiler/Codegen/LLVMGPU/test/conv_pipeline_test.mlir

```diff
@@ -1,4 +1,9 @@
-// RUN: iree-opt --split-input-file --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-lower-executable-target)))' %s | FileCheck %s
+// RUN: iree-opt --split-input-file \
+// RUN:   --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-lower-executable-target,canonicalize)))' \
+// RUN:   %s | FileCheck %s
+
+// Note: upstream hoisting on tensors does not canonicalize `scf.for` anymore,
```

Comment:
nit: I would move that to commit description I don't think it is a comment that needs to stay in the code.